src/Container/ServiceExtensions.php

```diff
@@ -111,7 +117,9 @@ protected function resolveByType(
 
         // 2nd group of extensions: targeting parent classes
         $parents = class_parents($service, false);
-        ($parents === false) and $parents = [];
+        if ($parents === false) {
+            $parents = [];
```

Comment:
Can we maybe add a test with a class that does neither extend a parent class, nor implement an interface, so we add coverage for this line and the one further down?

Reply:
I agree, but this PR is focused on swapping out the coding standards and I do not change the code here, just the formatting. We should create a follow-up PR with more UnitTests. 👍🏻